packages/swap/src/helpers/transformers.ts

```diff
@@ -152,7 +152,24 @@ export const transformersMaker = (
   }
 
   const asYoroiPools = (openswapPools: OpenSwap.Pool[]): Swap.Pool[] => {
-    return openswapPools?.length > 0 ? openswapPools.map(asYoroiPool) : []
+    const supportedProviders: Swap.SupportedProtocol[] = [
+      'minswap',
+      'wingriders',
+      'sundaeswap',
+      'muesliswap',
+      'muesliswap_v2',
+      'vyfi',
+    ]
+
+    return openswapPools?.length > 0
+      ? openswapPools
+          .filter((pool) =>
+            supportedProviders.includes(
+              pool.provider as Swap.SupportedProtocol,
```

Comment:
I'm a bit concerned about mixing `provider` and `protocol`.
Do openswapPools have a protocol? Otherwise, where are we getting the protocols from?

Reply:
yeah, could be misleading. The protocol/providerName we are getting it from the pool list entry. Maybe we can rename it as `SupportedProvider` but not sure

Reply:
yap, so I'm